src/Controls/tests/DeviceTests/Elements/Shell/ShellTests.cs

```diff
@@ -82,6 +82,7 @@ public async Task PageLayoutDoesNotExceedWindowBounds()
 				Assert.True(pageBounds.Height <= window.Height);
 			});
 		}
+#endif
```

Comment:
I think this line accidentally put `#if !MACCATALYST` across this entire file. 👀